packages/doc-contribs/pr-antora-content-guidelines-checker/src/main.ts

```diff
@@ -36,9 +41,18 @@ async function run(): Promise<void> {
     const forbiddenPatternToCheckInput = core
       .getInput("forbidden-pattern-to-check")
       .split(",");
+
+    const stepToSkip: string[] = core
+      .getInput("step-to-skip")
+      .split(",")
+      .map((item) => item.trim());
+
     let steps: Array<ValidationStep> = [];
 
-    if (core.getInput("attributes-to-check") !== "") {
+    if (
+      stepToSkip.includes(AvailableSteps.ATTRIBUTES_CHECKING_STEP) &&
```

Comment:
**nitpick:** It seems that the logic is inverted, we run a step when it is not skipped, so to me the condition should be 
```suggestion
      !stepToSkip.includes(AvailableSteps.ATTRIBUTES_CHECKING_STEP) &&
```